README.md

```diff
@@ -48,20 +51,24 @@ This is sometimes easier said than done.
 Sharing these research outputs means understanding data management, library sciences, software development, and continuous integration techniques: skills that are not widely taught or expected of academic researchers and data scientists. As these activities are not commonly taught, we recognise that the burden of requirement and new skill acquisition can be intimidating to individuals who are new to this world.
 _The Turing Way_ is a handbook to support students, their supervisors, funders and journal editors in ensuring that reproducible data science is "too easy not to do" even for people who have never worked in this way before.
 It will include training material on version control, analysis testing, and open and transparent communication with future users, and build on Turing Institute case studies and workshops.
-This project is openly developed and any and all questions, comments and recommendations are welcome at our github repository: <https://github.com/alan-turing-institute/the-turing-way>.
+This project is openly developed and any and all questions, comments and recommendations are welcome at our GitHub repository: <https://github.com/alan-turing-institute/the-turing-way>.
+
+### The Team
 
-### The team
+_The Turing Way_ a community-led project.
+Everyone who contributes to this book, no matter how small or big their contributions are, is recognised in this project as a contributor and a community member. 
+Long-term contributors of the project are considered part of the core contributors groups who take on various leadership roles in the project.
 
-This is (part of) the project team planning work at the Turing Institute.
-For more on how to contact us, see the [ways of working document](ways_of_working.md).
+The project is coordinated by the lead investigators **Kirstie Whitaker** (founder) and **Malvika Sharan** (community developer) and hosted at [The Alan Turing Institute](https://www.turing.ac.uk/).
 
-![Team photo](book/website/figures/theturingway-team.jpg)
+You can read [_The Turing Way_ acknowledgement process](https://the-turing-way.netlify.app/community-handbook/acknowledgement/acknowledgement-members.html#community-members-contributors-and-co-authors) and [Record of Contributions](https://the-turing-way.netlify.app/afterword/contributors-record.html) to learn about how we acknowledge your work and how our contributors are highlighted in the project.
+You can see the [Contributors Table](#contributors) to see the GitHub profiles of all our contributors.
 
 ### Contributing
 
```

Comment:
![There are six illustrations to show six types of contributions that anyone can make. These are 1)developed and share, Maintain and improve, share resources, review and update, make it global through translation, share best practices](book/website/figures/README_imgs/README_contributions.png)

Reply:
The contributing guidelines are VERY good but very long - I think having something very brief added to the README can be powerful in attracting more contributors. I added the slides about the six types of contributions you can make!  
BTW: you can always make up the same slide by combining images in Markdown but it will take me time to polish it exactly as the slide so I just exported the slides as images and added them in a new folder `README_imgs`

Reply:
These are amazing additions. I think we can integrate the introductory 5 minutes slides using HTML (iframe) and keep this as the first slide?

Reply:
Thank you Malvika, do you think it's better if the slides have a dark or white background?

Reply:
I think white background will work well in this case.

Reply:
I have changed your comment as a suggestion. Please go ahead and commit this when you are ready.

Reply:
Unfortunately, the iframe option for sharing [slides](https://docs.google.com/presentation/d/e/2PACX-1vSK-zDTKraDEbdMBJG-XHmMLdkcfKMR9v4Hca1tXkQwdqsWidLQd8LNArUy-xjM0X8sdAeaR90Q2-zD/pub?start=false&loop=false&delayms=3000) did not work on this markdown file (works on HackMD which made me think that it will work here as well). I am ok to not have the slides. Sharing the tag in case you know any trick:
```
<iframe src="https://docs.google.com/presentation/d/e/2PACX-1vSK-zDTKraDEbdMBJG-XHmMLdkcfKMR9v4Hca1tXkQwdqsWidLQd8LNArUy-xjM0X8sdAeaR90Q2-zD/embed?start=false&loop=false&delayms=3000" frameborder="0" width="960" height="569" allowfullscreen="true" mozallowfullscreen="true" webkitallowfullscreen="true"></iframe>
```

Reply:
Interesting! From GitHub [specs](https://github.github.com/gfm/#example-630), it says iframe is one of the disallowed Raw HTML (extension) in GitHub flavoured Markdown (GFM). 
> the following HTML tags will be filtered when rendering HTML output:
> <title><textarea><style><xmp><iframe>